compliance-web/src/components/App/Inspections/InspectionDrawer.tsx

```diff
@@ -62,13 +62,9 @@ const InspectionDrawer: React.FC<InspectionDrawerProps> = ({
   const { data: attendanceList } = useAttendanceOptionsData();
   const { data: agenciesList } = useAgenciesData();
   const { data: firstNationsList } = useFirstNationsData();
+  const { data: staffUserList } = useStaffUsersData()
```

Comment:
we need to use a different variable for this, may be like inattendanceOfficersList, and use it for inattendance officers dropdown.
this staffUserList must be same as before, and use it for primary user list.